pkg/systemd/overview-cards/lastLogin.tsx

```diff
@@ -53,10 +53,17 @@ const getFormattedDateTime = (time) => {
     return timeformat.dateTime(date);
 };
 
+type LoginMessages = {
+    "last-login-time": number,
+    "last-login-host": string,
+    "last-login-line": string,
+    "fail-count"?: number,
```

Comment:
Why can this be undefined? In cockpit session it is initialized as 0

Reply:
No, it doesn't get written if fail_count == 0, see https://github.com/cockpit-project/cockpit/blob/main/src/session/session-utils.c#L338

Reply:
But we only print when we call `cockpit_json_print_integer_property`? Otherwise we `goto out`.